package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@pkgjs/support",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "description": "Package support information (see: https://github.com/nodejs/package-maintenance/pull/220)",
-  "author": "Wes Todd <wes@wesleytodd.com>",
+  "author": "pkgjs contributors - https://github.com/pkgjs/support/graphs/contributors",
```

Comment:
does this work?  I think we will want to use the contributors field and still have a normal author.  I am pretty sure npm uses this to link to publishers account, right?

Reply:
oh right, this doesn't work. "author" has to be the same as the contents of "contributors" - a name, with an optional email and optional URL.

Reply:
that said, no, there's no link here to any npm accounts, this is just not valid according to the schema for package.json.

Reply:
probably not, should I leave the author as is for now ?

Reply:
ok just removed that, want to be ready to get the blog post out and need a publish for that, we can figure out the author field later on.

Reply:
Yeah, that makes sense at first.  I think we should discuss the general strategy here for the org, but it seems to me we just add each new contributor ad-hoc?